NOTICE.md

```diff
@@ -1,6 +1,8 @@
 HASH: 152b6b858d2f6b94fa398fa4545d5e130ab24434
 #############################################################
-WARNING: This update bumps to rpi-4.19.y linux tree
+WARNING: This update bumps to a pre-release rpi-4.19.y linux
+tree and should only be used if there is a specific reason to 
+do so, for example, a request by a Raspberry Pi engineer.
```

Comment:
`do so - for example, ...`

More importantly, I think we need to separate the standard "rpi-update is dangerous, mmkay" message from the specific warning about the pre-release tree. Which isn't, anymore.

Reply:
"This update bumps to a untested rpi-4.19.y linux tree."  <<< not entirely sure what adjective to use.
""
"You should only use rpi-update if there is a specific reason to   <<< separate paraghraph to emphasise.
do so, for example, a request by a Raspberry Pi engineer."

Reply:
Pre-release was the correct phrase, but now the clause should be removed, making the separate paragraph for the usual warning even more important.

Reply:
Yes, remove reference to 4.19.y.
That is no longer relevant and reference to kernel version should only be present when rpi-update kernel is on a never version than apt kernel.

Reply:
I've made it even more obvious, and removed any version number references.